Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java

```diff
@@ -217,7 +217,11 @@ protected void createOrOverwrite(CopierData data, LegacyDocStructHelperInterface
      * of the path aren’t available. This works recursively, by calling itself
      * on the subnode, if found, or returning null otherwise.
      *
+<<<<<<< HEAD
```

Comment:
you inserted merge-conflict code

Reply:
Seems to be gone with the latest rebase.

Reply:
Seems not be gone wit the latest rebase - still visible in the file view: https://github.com/kitodo/kitodo-production/blob/ab049a98d126acd2eef0d8587a5bae27173edfb7/Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java